lightstep/resource_metric_condition.go

```diff
@@ -966,7 +982,15 @@ func buildKeys(keysIn []interface{}) []string {
 func buildThresholds(singleExpression map[string]interface{}) (client.Thresholds, error) {
 	t := client.Thresholds{}
 
-	thresholdsObj := singleExpression["thresholds"].([]interface{})[0].(map[string]interface{})
+	elem, ok := singleExpression["thresholds"]
```

Comment:
previously `thresholds` was marked as required so we could count on there being a non-empty list, but now it's optional so we can't